docs/docs/configuration/live.md

```diff
@@ -24,6 +24,8 @@ webRTC works by creating a websocket connection on extra ports. One of the follo
     * For local webRTC, you will need to create your own go2rtc config:
 
 ```yaml
+ffmpeg:
+  bin: /usr/lib/btbn-ffmpeg/bin/ffmpeg # <--- this is required
```

Comment:
Does this change or do we need to add a symlink to this path for arm32?

Reply:
Actually is this really required? If there is no other ffmpeg executable in the container, I don't see why it's required.

Reply:
It definitely would be broken either way for 32bit with dev or this PR. arm32 support is already on the outs so will need to confirm if it's planned for 0.12

Reply:
> Actually is this really required? If there is no other ffmpeg executable in the container, I don't see why it's required.

Yes it is required, without it go2rtc does not find the executable in the path

Reply:
I think I know why...

Reply:
Try changing:

https://github.com/blakeblackshear/frigate/blob/c97aac6c94cc8888864b3e6c45b24969f45a78c7/docker/rootfs/etc/services.d/go2rtc/run#L1

To:

```
#!/command/with-contenv bash
```

And actually you should refactor it for all the `run` scripts. By default (when you do not use `with-contenv`, s6-overlay will cleanup all the environment variables you have set, including PATH, before starting the service.

Reply:
BTW go2trc bin path is not in PATH, I'd recommend you add it to PATH as well.

Reply:
> Try changing:
> 
> https://github.com/blakeblackshear/frigate/blob/c97aac6c94cc8888864b3e6c45b24969f45a78c7/docker/rootfs/etc/services.d/go2rtc/run#L1
> 
> To:
> 
> ```
> #!/command/with-contenv bash
> ```

That's not working, I get:

```
s6-supervise (child): fatal: unable to exec run: No such file or directory
s6-supervise go2rtc: warning: unable to spawn ./run - waiting 10 seconds
```

Reply:
Oh! I thought you were using s6-overlay v3 but you are using v2. I'll send a PR updating to v3.

Reply:
@NickM-27, meanwhile, with s6 v2, you need to change `/command/with-contenv` with `/usr/bin/with-contenv`.

Reply:
I'll try that for now, will update accordingly once #4393 is resolved

Reply:
Cool that worked, so that should solve the issues for 32 bit arm as well.

Reply:
We can keep using s6-overlay 2.x for now. There's no specific gain on upgrading it at this point.